### PR TITLE
task1 files upload

### PR DESCRIPTION
**Pull request by alekseysushkevich**

### CODE REVIEW

File: .DS_Store
Comment (dmitryafanasievissoft) [inline .DS_Store:1]: DS_Store should be included into gitignore file

File: task1/task1.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXFrameworksBuildPhase section */
+
+/* Begin PBXGroup section */
+		FB7E03612CDBA4D500C91013 = {
+			isa = PBXGroup;
+			children = (
+				FB7E036C2CDBA4D500C91013 /* task1 */,
```
Comment (dmitryafanasievissoft) [inline task1/task1.xcodeproj/project.pbxproj]: I don't think you want to name your project "task1". Pick up a proper name

File: .DS_Store
Comment (alekseysushkevich) [reply .DS_Store:1]: fixed

File: task1/task1.xcodeproj/project.pbxproj (hunk 1 above)
Comment (alekseysushkevich) [reply task1/task1.xcodeproj/project.pbxproj]: fixed

**Review by dmitryafanasievissoft — APPROVED** [review]

**Pull request closed without merging**